packages/shared/src/components/auth/LoginForm.tsx

```diff
@@ -119,13 +120,9 @@ function LoginForm({
           <AlertParagraph className="flex-1 !mt-0">
             The email or password you entered doesn&apos;t match our records.
             Please try again or{' '}
-            <button
-              type="button"
-              onClick={onSignup}
-              className="font-bold underline"
-            >
+            <ClickableText onClick={onSignup} className="font-bold underline">
```

Comment:
I think this should use `inverseUnderline` prop of ClickableText
Although not 100% sure if it adds more unwanted styles maybe?

Reply:
If I use `inverseUnderline`, it is not underlined by default and only on hover. I think we want the opposite based on the current implementation. I changed it so that we underline, but remove underline on hover. This is aligned with the behavior of other similar buttons on the login forms.

This is how it looks now:
<img width="395" alt="Screenshot 2023-12-22 at 10 46 24" src="https://github.com/dailydotdev/apps/assets/9974711/fcd77727-5e60-4d88-b4db-9d9e4b860d46">